app/models/miq_alert.rb

```diff
@@ -844,4 +845,23 @@ def self.import_from_yaml(fd)
       stat
     end
   end
+
+  def hawkular_keys
+    aux = super
```

Comment:
aux is the data in hawkular_keys column, whose getter I'm overriding to never return `nil`.